feat: add content_sid variable (#18164)
supabase/supabase: apps/docs/pages/guides/auth/phone-login/twilio.mdx

Now the backend should be setup, we can proceed to add our client-side code!

<Admonition type="info">
  The `Twilio Content SID` field is specific to Twilio Programmable Messaging (WhatsApp) senders.

>   You may disregard this field if you are only sending SMS messages. Refer to the section on

Reviewer: can remove "you may"
just:
"Disregard this field if..."

>   You may disregard this field if you are only sending SMS messages. Refer to the section on

Author: ```suggestion
Disregard this field if you are only sending SMS messages. Refer to the section on
```

Complete the following steps to use WhatsApp OTP with Twilio Programmable Messaging:

1. Go through the [Twilio self sign up guide for WhatsApp](https://www.twilio.com/docs/whatsapp/self-sign-up).
2. Follow the [Twilio Guide for Submitting Templates](https://www.twilio.com/docs/whatsapp/tutorial/send-whatsapp-notification-messages-templates#creating-message-templates-and-submitting-them-for-approval) and submit a Basic Authentication Message for approval.

Alternatively, you can register a Twilio Content SID which corresponds to a message template.
![Twilio Content SID Image](/docs/img/guides/auth-twilio/twilio_content_sid.png)

> Note that you may only use one Twilio Content SID at a time. Consequently, we recommend using Twilio Verify or the Basic Authentication Message if you require internationalization on your message template.

Reviewer: maybe don't need "consequently" here

> Note that you may only use one Twilio Content SID at a time. Consequently, we recommend using Twilio Verify or the Basic Authentication Message if you require internationalization on your message template.

Author: ```suggestion
Note that you may only use one Twilio Content SID at a time. Use Twilio Verify or the Basic Authentication Message if you require internationalization for your message template.
```
